README.md

```diff
@@ -35,3 +35,7 @@ When a new valid value needs to be added to the data model:
 
 4. When ready to release a new version of the data model and a pull request is made, the valid values will automatically get added to `mc2.model.csv` and its JSON-LD file.
 
+## Release Notes 
+
+The latest version release is v23.12.1. Moving forward, this repository will adopt <a href="https://semvar.org/" target="_blank"> Semanting Versioning</a>  (SemVar), which provides a standard way of versioning software based on meaningful changes in functionality.
```

Comment:
```suggestion
The version release tag for January 2024 is v23.12.1. Moving forward, this repository will adopt <a href="https://semvar.org/" target="_blank"> Semanting Versioning</a>  (SemVar), which provides a standard way of versioning software based on meaningful changes in functionality.
```

Reply:
I added a small edit. Also, the link here is not going to the right place. Could you please check the link?